api/src/test/java/org/openmrs/util/DatabaseUpdaterTest.java

```diff
@@ -9,13 +9,25 @@
  */
 package org.openmrs.util;
 
+import org.junit.After;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.openmrs.test.BaseContextSensitiveTest;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import liquibase.exception.LockException;
 
+import java.awt.*;
```

Comment:
We generally discourage using wild card imports https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
fixed